src/components/ContentSidebar/ActivityFeed/activity-feed/activityFeedUtils.js

```diff
@@ -45,15 +50,17 @@ export function collapseFeedState(feedState: ?FeedItems): FeedItems {
                 [prevModifiedBy.id]: { ...prevModifiedBy },
```

Comment:
Should we just exclude collaborators without a valid user object?

Reply:
I tried this out and it seems misleading because if we have 2 collaborators and 1 is null, then the UI will show that "<name of collaborator> uploaded versions x - y". Will leave it for now and we can address this as part of the follow on work to clean up this code/UX